### PR TITLE
AZ-551: Add CLI tool for key and validator rotation

### PR DESCRIPTION
**Pull request by timorl**

### CODE REVIEW

File: bin/cliain/src/validators.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+use log::info;
+use aleph_client::{change_members, create_connection, KeyPair};
+use substrate_api_client::{AccountId, XtStatus};
+use sp_core::crypto::Ss58Codec;
+
+/// Change validators to the provided list by calling the provided node.
+/// The keypair has to be capable of signing sudo calls.
+pub fn change(validators: Vec<String>, node: String, key: KeyPair) {
```
Comment (Marcin-Radecki) [inline bin/cliain/src/validators.rs:8]: We can change the name `key` to `sudo_key` to reflect this function usage better.

Comment (Marcin-Radecki) [inline bin/cliain/src/validators.rs:8]: We can change the name `key` to `sudo_key` to reflect this function usage better.

Comment (timorl) [reply bin/cliain/src/validators.rs]: Done.

**Review by Marcin-Radecki — APPROVED** [review]

**Review by deuszx — COMMENTED** [review]
Do you think it's worth adding a README or some kind of descriptive `--help` message to the new command?

**Review by maciejnems — APPROVED** [review]
LGTM, only one stupid nit

File: bin/cliain/rustfmt.toml [hunk 2]
```diff
@@ -0,0 +1 @@
+edition = "2021"
```
[the remark this answers is not in the document]
Comment (timorl) [reply bin/cliain/rustfmt.toml]: Done.

**Comment by timorl** [comment]
Also added some minimal documentation to the command that was lacking it @deuszx . Probably insufficient for informed external usage, but I believe we will keep developing this tool and we should build up a better documentation on the way when more commands appear and the interface becomes more standardized.

**Pull request merged**